src/Update.cpp

```diff
@@ -23,6 +23,10 @@ void LatentToInfectious(int cellIndex);
 void InfectiousToRecovered(int cellIndex);
 void InfectiousToDeath(int cellIndex);
 
+// if the dest cell state == src cell state, use this
```

Comment:
Can we have some more detailed comments on the parameters here please?  (Use doxygen formatting - I'd like to slowly add that as we go along - and VS 16.6 now supports that in tooltips).

Reply:
Done.